ci/README.md

```diff
@@ -0,0 +1,67 @@
+#### What is functional test:
+
+Functional CI is about running functional tests in ci.centos.org for the whole build pipeline
+to check its end to end functionality. This aims to check behavior for all the stages in the 
+pipeline and matches with expected one. If there is any difference in expected and actual status
+of the stages, CI fails and prints all the logs for the pipeline run.
+
+#### Purpose:
+
+This is to test the end to end functionality of the service before deploying it to some infra. 
+CI aims to make sure for an early feedback loop and runs on each push to the repository.
+
+#### How it works:
+![Container Pipeline Functional CI Diagram](../docs/diagrams/functional_ci.png)
```

Comment:
The diagram is referencing the openshift brach, can you update it to reflect the scenario for master branch?

Reply:
As it would be once the code moves to master branch of container-pipeline-service

Reply:
@navidshaikh I was thinking to update the diagram once we move to master branch. Because master branch is having a different code base now, descriptions wont match with current scenario

Reply:
Upstream pipeline service's master branch consists of entirely different code and architecture. I'd suggest we keep it as is right now and be quick to deprecate upstream master branch and replace it with openshift branch.